frontend/src/scene/main-view/CourseModal.js

```diff
@@ -104,7 +106,48 @@ const ReservationContent = styled(Content)`
     }
 `;
 
-const MainModal = ({ course, seletectedDate, onConfirm, clear }) => (
+const ErrorMessageAnimation = posed.h4({
+    hidden: {
+        y: -10,
+        x: 50,
+        opacity: 0,
+    },
+    shown: {
+        y: -10,
+        x: 0,
+        opacity: 1,
+    },
+});
+
+const ErrorMessage = styled(ErrorMessageAnimation)`
+    text-align: center;
+    font-size: 2.5rem;
+    color: white;
+    opacity: 0;
+    position: absolute;
+    top: 30%;
+    left: 0;
+    width: 100%;
+    z-index: 10;
+    * {
+        color: inherit !important;
+    }
+`;
+
+const ErrorMessageTag = styled(ErrorMessageAnimation)`
```

Comment:
We dont need `ErrorMessageAnimation` since we are no using any animations here. You can change it to `posed.div` and remove the `ErrorMessageAnimation` object.

Reply:
`ErrorMessageAnimation` is needed here for the sake of displaying message.